app/controllers/ping_controller.rb

```diff
@@ -1,10 +1,10 @@
-class PingController  < ApplicationController
+class PingController < ApplicationController
   def index
     render json: {
-      :version_number => VERSION_INFO['version_number'],
-      :build_date => VERSION_INFO['build_date'],
-      :commit_id => VERSION_INFO['commit'],
-      :build_tag => VERSION_INFO['build_tag']
+      version_number:  VERSION_INFO['version_number'],
+      build_date:      VERSION_INFO['build_date'],
+      commit_id:       VERSION_INFO['commit'],
+      build_tag:       VERSION_INFO['build_tag']
```

Comment:
I gots me a plugin to do it real purty like :)

On 25 September 2015 at 14:55, Steve Marshall <notifications@github.com>
wrote:

> In app/controllers/ping_controller.rb
> <https://github.com/ministryofjustice/prison-visits/pull/332#discussion_r40431387>
> :
>
> >    def index
> >      render json: {
> > -      :version_number => VERSION_INFO['version_number'],
> > -      :build_date => VERSION_INFO['build_date'],
> > -      :commit_id => VERSION_INFO['commit'],
> > -      :build_tag => VERSION_INFO['build_tag']
> > +      version_number:  VERSION_INFO['version_number'],
> > +      build_date:      VERSION_INFO['build_date'],
> > +      commit_id:       VERSION_INFO['commit'],
> > +      build_tag:       VERSION_INFO['build_tag']
>
> [image: :+1:]
>
> Lining up FTW.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/prison-visits/pull/332/files#r40431387>
> .
>